README.md

```diff
@@ -76,6 +91,52 @@ would need to aggregate with key `[game, score]`.
 To support different sorting and partitioning keys, you can define multiple
 instances. See [below](#defining-multiple-aggregates) for details.
 
+If you separate your data via the `sortKey` and `prefix` bounds, you can look at
+your data from any altitude. You can do a global `count` to see how many total
+data points there are, or you can zero in on an individual group of the data.
+
+However, there's a tradeoff: nearby data points can interfere with each other
+in the internal data structure, reducing throughput. See
+[below](#read-dependencies-and-writes) for more details. To avoid interference,
+you can use Namespaces.
+
+### Namespacing
+
+If your data is separated into distinct partitions, and you don't need to
+aggregate between partitions, then you can put each partition into its own
+namespace. Each namespace gets its own internal data structure.
+
+If your app has multiple games, it's not useful to aggregate scores across
+different games. The scoring system for chess isn't related to the scoring
+system for football. So we can namespace our scores based on the game.
+
+Whenever we aggregate scores, we *must* specify the namespace.
+On the other hand, the internal aggregation data structure can keep the scores
+separate and keep throughput high.
+
+Here's how you would create the aggregate we just described:
+
+```ts
+const leaderboardByGame = new TableAggregate<{
+  namespace: Id<"games">,
+  key: number,
+  dataModel: DataModel,
+  tableName: "scores",
```

Comment:
```suggestion
  Namespace: Id<"games">,
  Key: number,
  DataModel: DataModel,
  TableName: "scores",
```

Reply:
and I'm not sure if that allows you to do just `DataModel,`

Reply:
It does not allow that. While i'm find-replacing, i was wondering if Key should be called SortKey